lms/djangoapps/courseware/views/index.py

```diff
@@ -715,9 +719,9 @@ def show_courseware_mfe_link(user, staff_access, course_key):
     # course team preview CourseWaffleFlag for this course *or* if we've turned
     # on the redirect for your students.
     mfe_enabled_for_course_team = COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW.is_enabled(course_key)
-    mfe_enabled_for_students = should_redirect_to_courseware_microfrontend(course_key)
+    mfe_experiment_enabled_for_course = REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled_for_course(course_key)
```

Comment:
You're still importing and using `should_redirect_to_courseware_microfrontend` elsewhere. Should those instances be switched to check the waffle too?

Reply:
In those `should_redirect_to_courseware_microfrontend ` cases we want to know if this user is in the treatment group. The `should_redirect_to_courseware_microfrontend` function uses `REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled()`. 

In this case we want to check if the experiment is on for this course.